projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.scss

```diff
@@ -6,8 +6,12 @@
 }
 
 .mat-dialog-container{
-  padding: 12px;
+  padding: 5px;
   border-radius: 36px;
   background-color: var(--theme-blue-button-color);
   color: var(--theme-about-panel-background-color);
 }
+
+.open-in-app {
+  overflow-x: hidden !important;
```

Comment:
do you need !important? Can't you change somehow your selector used in scss so it won't need !important?

Reply:
yess i checked without !important as this style is applied to button from angular material but without !important it will not override the mat style.

Reply:
Did you try something like: 
.material-css-selector.open-in-app {
  ....
}

You may need to combine your css class with material class which is applied to button to override it without important

Reply:
it is not applying while i am doing like this, so can you please just check

Reply:
Not sure why it is not applying I checked below on some button: 
<img width="630" alt="image" src="https://user-images.githubusercontent.com/115619721/219689547-70da6ead-744f-493f-8bf5-7045c00caa9d.png">


You see I added overflow-x: hidden to 'test' class which I applied to button and then in computed I can see it is changed: 
<img width="610" alt="image" src="https://user-images.githubusercontent.com/115619721/219689867-32af5fdd-50a0-4660-a5fd-6cc971f5bda2.png">


but as I said, I tried it on different button

Reply:
It is not applying in my case, don't know why, I also checked with material-css-selector

![Screenshot from 2023-02-17 20-47-30](https://user-images.githubusercontent.com/105338943/219693677-1c7c78f4-0a45-414d-a27a-56d21a42b760.png)

Reply:
Just to ensure. When you are saying that you checked with "material-css-selector" do you mean that you used "material-css-selector" sentence in your styles or you replaced it with existing material class? I said "material-css-selector" as placeholder for real material class so it should be something like: 

.open-in-app.mat-button {
    overflow-x: hidden;
}

Reply:
yess you are right, I now applied the desired style and it's working now :)

Reply:
thank you